README.md

```diff
@@ -25,6 +25,8 @@ data to the framework.
     aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}} # required
     aws-session-token: ${{secrets.AWS_SESSION_TOKEN}} # required
     report-path: ./d2l-test-report.json # optional, defaults to shown path
+    lms-build-number: '20.24.1.123456' # optional
+    lms-instance-url: https://cd2024112345.devlms.desire2learn.com # optional
```

Comment:
I'm thinking it might be a good idea to sanitize this URL. Right now I don't even validate it. I can do that in another PR. Mainly thinking about if someone passes in `https://cd2024112345.devlms.desire2learn.com` vs `https://cd2024112345.devlms.desire2learn.com` I'd like those to look the same in the backend data.